devops/scripts/rebase-develop.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# If current branch is not master or doesnt start with release...
+if [[ "$CIRCLE_BRANCH" != "master"  && "$CIRCLE_BRANCH" != release*  ]]; then
```

Comment:
Hmm, wouldn't it be dandy if this [`$CIRCLE_TARGET_BRANCH` environmental variable feature request](https://discuss.circleci.com/t/create-a-circle-target-branch-envar/10022) were implemented. But this is fine, we'll just need to remember that PRs won't be rebasing on the target branch for `master` and `release*`.